Bottom: 379a8b9fece13edd6ae6bbc8c942029c10d46bf9
Top:    e577519bc31bf0d73cc759e2742ddcbda66bf242
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Remove container conf directory

Roles create it themselves


---
diff --git a/roles/docker-common/tasks/main.yml b/roles/docker-common/tasks/main.yml
index 396b842c..7ef3f40c 100644
--- a/roles/docker-common/tasks/main.yml
+++ b/roles/docker-common/tasks/main.yml
@@ -8,13 +8,6 @@
 - import_tasks: sshd_settings.yml
 - import_tasks: mariadb_settings.yml
 
-- name: base docker_container.conf dir
-  file:
-    state: directory
-    path: '{{ docker_restore_config_base_dir }}'
-    owner: root
-    group: root
-    mode: 'u=rwx,g=rx,o=rx'
 
 
 - name: Check if baculavirt selinux module exists
